src/components/student/CourseEnroll.js

```diff
@@ -1,5 +1,7 @@
 import React, {useState, useEffect} from 'react';
```

Comment:
Including my changes in this PR will probably create a merge conflict. I would revert the changes I made.